src/SourceLocator/Type/FileSystemIteratorSourceLocator.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace BetterReflection\SourceLocator\Type;
+
+use BetterReflection\Identifier\Identifier;
+use BetterReflection\Identifier\IdentifierType;
+use BetterReflection\Reflector\Reflector;
+use BetterReflection\SourceLocator\Exception\InvalidFileInfo;
+
+/**
+ * This source locator loads all php files from \FileSystemIterator
+ */
+class FileSystemIteratorSourceLocator implements SourceLocator
+{
+    /**
+     * Don't touch this variable directly
+     * @var AggregateSourceLocator
+     */
+    private $aggregatedSourceLocator;
```

Comment:
Naming should be `$aggregateSourceLocator` to be consistent (aggregated vs aggregate) please